rotkehlchen/api/rest.py

```diff
@@ -4624,12 +4633,17 @@ def delete_addressbook_entries(
                 status_code=HTTPStatus.CONFLICT,
             )
 
-    def search_for_names_everywhere(self, addresses: list[ChecksumEvmAddress]) -> Response:
+    def search_for_names_everywhere(
+            self,
+            addresses: list[tuple[ChecksumEvmAddress, Optional[SupportedBlockchain]]],
+    ) -> Response:
         mappings = search_for_addresses_names(
             database=self.rotkehlchen.data.db,
             addresses=addresses,
         )
-        return api_response(_wrap_in_ok_result(mappings))
+        return api_response(_wrap_in_ok_result(process_result_list(
+            [[*x] for x in mappings],
+        )))
```

Comment:
This is changing the response format right? It should update also the docs. Also have you talked with the frontend to discuss what format is better for them or if this one makes sense? 

Lastly don't use single letter variables, here is confusing and is not clear what `x` is just by reading this part of the code. Use something like `named_address_tuple` or something more descriptive. In other places by the context is easy but here it comes from a function call that is already complex

Reply:
Yeah, thank you for reminding to ask frontend guys :pray: 

Didn't change the docs yet because wanted to get an agreement on the api format first